### DISCUSSION TITLE
IPED-SearchApp.exe not launching

### DISCUSSION THREAD

**Discussion by Phyambre** [sepinf-inc/IPED#2368] [General]
Hello,
I downloaded IPED-4.1.6_plus_java_plugins.zip  from releases. Then I uncompressed the file.
Now, I am trying to open IPED-SearchApp.exe 
When I double click the executable, I just receive the following error:
![iped_error](https://github.com/user-attachments/assets/aaed0772-2c3a-4f5d-ba80-df002a04420a)

It doesn't make any sense, because the lib folder by default is not in that path. If I create that folder and copy all the files from lib folder there, then the splash screen of IPED-SearchApp.exe is shown, but then it dies. If I try to launch the exe file from powershell, nothing is written in the console. The application just dies.
OS is Windows 11 and the JVM is Bellsoft JDK 11 (I also installed JDK 23 but it is not working with that).
Best regards,
Phy.

**Comment by wladimirleite** [comment]
IPED-SearchApp is to be used to open a processed case.
Have you used iped.jar (or iped.exe) to process an evidence and create a case?

**Comment by Phyambre** [comment]
No, I have just unpacked the archive and I was trying to check that the exe files worked correctly.

**Reply by wladimirleite** [reply]
When you process a case (can be just a test folder with a few files), IPED will build the correct folder structure in the case destination folder. The search app will work there...

**Comment by Phyambre** [comment]
Thank you!

**Comment by sandovalcabral** [comment]
Ao processar um caso, tentar abrir o IPED-SearchApp e dar o seguinte erro: "Erro: não foi possível encontrar java.dll". Depois aparece o seguinte erro:  "Erro: Não foi possível encontrar o Java SE Runtime Environment" . E agora?
